config/couchdb/nouveau.ini

```
@@ -0,0 +1,2 @@
+[nouveau]
+enable = true
```

Just one change needed.

```suggestion
[nouveau]
enable = true
url = http://couchdb-nouveau:5987
```